util/viz-list.ts

```diff
@@ -2710,5 +2710,20 @@ export const vizList: VizItem[] = [
         tools: undefined,
         luminosity: ["light"],
         interactive: true
-    }
+    },
```

Comment:
{
    id: 184,
    title: "CO2 emissions in Europe",
    date: new Date(2024, 7),
    author: "Joseph Barbier",
    url: "https://python-graph-gallery.com/web-map-with-custom-legend/",
    img: [
        { full: "Choropleth-map-with-a-customized-legend.png", zoom: "Choropleth-map-with-a-customized-legend-zoom.png", chartId: ["map"] ["choropleth"]},
    ],
    contextDescription: "Europe has been actively working to reduce CO2 emissions, with many countries implementing diverse strategies based on their economic structures and policies. This has resulted in varying levels of emissions across the region.",
    chartDescription: "The Choropleth map features a well-crafted legend that guides viewers through different levels of CO2 emissions. It includes clear annotations and uses a vibrant color scheme, making it both engaging and informative.",
    tools: undefined,
    luminosity: ["light"],
    interactive: true
    },

Reply:
Please review and let me know any further changes if required. @holtzy

Reply:
Hi @Shirisha-18 

Thanks for the update. However, you need to update the code in the Pull request, not provide it as a comment as above! Otherwise I cannot merge it.

Also, the images are still not included in the pull request. Can you add them?

Reply:
Hi @holtzy,

I have updated the code in PR. I uploaded images yesterday, but I believe I did it incorrectly. I will correct this by EOD, as I don't have my laptop with me right now.